### PR TITLE
Add support for non label-schema labels

### PR DESCRIPTION
**Pull request by The-Loeki**
After the label-schema.org label support was added, this PR now adds the strongly suggested `org.label-schema.version` label by default and adds support for 'regular' labels similar to the tags and build args.
This incidently solves both comments in the original PR ;) 

For backwards compatibility I've kept the `label-schema` option.

# References
#134 
#135

### CODE REVIEW

**Comment by bradrydzewski** [comment]
`Labels` is a reserved keyword in the yaml, which means that `PLUGINS_LABELS` will not be populated and sent down to the plugin. Instead you should use a different name, such as `image_labels` in the yaml with the corresponding `PLUGINS_IMAGE_LABELS` environment variable.

File: cmd/drone-docker/main.go [hunk 1]
```diff
@@ -171,6 +171,11 @@ func main() {
 			Usage:  "docker repository",
 			EnvVar: "PLUGIN_REPO",
 		},
+		cli.StringSliceFlag{
+			Name:   "Labels",
+			Usage:  "labels",
+			EnvVar: "PLUGIN_LABELS",
```
Comment (bradrydzewski) [inline cmd/drone-docker/main.go]: `labels` in the yaml is a reserved keyword, so we'll need to choose another attribute name

Comment (The-Loeki) [reply cmd/drone-docker/main.go]: I figure only `!label` is reserved?

Comment (bradrydzewski) [reply cmd/drone-docker/main.go]: container labels are reserved per the docker-compose specification. See https://docs.docker.com/compose/compose-file/#labels-1. Drone therefore uses these fields for special purposes

Comment (bradrydzewski) [reply cmd/drone-docker/main.go]: note that `dns` is also a reserved attribute, so you will notice it uses `custom_dns` with the `custom_` prefix. I would therefore recommend we follow the same pattern with `custom_labels`

Comment (The-Loeki) [reply cmd/drone-docker/main.go]: Is this what you had in mind ;)?

**Comment by The-Loeki** [comment]
@bradrydzewski can you merge this or was there something else you wanted me to fix?

**Comment by johnrengelman** [comment]
@bradrydzewski any additional feedback on this?
I'm looking at adding the build number label as we discussed at NeXT and figured I could get this across the finish line too.

**Comment by bradrydzewski** [comment]
sorry for the delay, merged

**Pull request merged**